src/lib/items/Item.js

```diff
@@ -143,7 +143,7 @@ export default class Item extends Component {
 
       for (var key of Object.keys(groupTops)) {
         var item = groupTops[key]
-        if (e.pageY - topOffset + scrollOffset > item) {
```

Comment:
This took me a while to find and led me down different paths to a fix before I realized this bit of logic is simply overcomplicated. As long as we recalculate the top of the timeline frequently, we don't need to track the scroll containers' states.